styles.css

```diff
@@ -0,0 +1,78 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@500;700&display=swap');
+
+* {
+  margin: 0;
+}
+
+.toolbar {
+  display: flex;
+  justify-content: space-between;
+  background-color: #3c3a39;
+  width: 100%;
+  align-items: center;
+}
+
+#my-logo {
+  color: #fff;
+  font-size: 18px;
+  line-height: 24px;
+  padding-left: 24px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+}
+
+.menu-bar {
+  color: #acb7c3;
+  font-size: 36px;
+  padding-right: 24px;
+  margin-top: 19px;
+  margin-bottom: 19px;
+}
+
+.headline-section {
+  width: 100%;
+  padding: 139px 0;
+  background-color: #1c1a19;
+  white-space: normal;
+  position: relative;
+}
```

Comment:
As part of the project requirements and for best CSS practices, kindly align the contents of the headline section with flexbox. 👍